web/web.go

```diff
@@ -0,0 +1,771 @@
+	stmtstats = stmtstats + "  schema: statements successfully processed for Spanner schema information.\n"
+	stmtstats = stmtstats + "    data: statements successfully processed for data.\n"
+	stmtstats = stmtstats + "    skip: statements not relevant for Spanner schema or data.\n"
+	stmtstats = stmtstats + "   error: statements that could not be processed.\n"
+	stmtstats = stmtstats + "  --------------------------------------\n"
+	stmtstats = stmtstats + fmt.Sprintf("  %6s %6s %6s %6s  %s\n", "schema", "data", "skip", "error", "statement")
+	stmtstats = stmtstats + "  --------------------------------------\n"
+	for _, x := range l {
+		s := conv.Stats.Statement[x.statement]
+		stmtstats = stmtstats + fmt.Sprintf("  %6d %6d %6d %6d  %s\n", s.Schema, s.Data, s.Skip, s.Error, x.statement)
+	}
+	if driverName == "pg_dump" {
+		stmtstats = stmtstats + "See github.com/lfittl/pg_query_go/nodes for definitions of statement types\n"
+		stmtstats = stmtstats + "(lfittl/pg_query_go is the library we use for parsing pg_dump output).\n"
+		stmtstats = stmtstats + "\n"
+	} else if driverName == "mysqldump" {
+		stmtstats = stmtstats + "See https://github.com/pingcap/parser for definitions of statement types\n"
+		stmtstats = stmtstats + "(pingcap/parser is the library we use for parsing mysqldump output).\n"
+		stmtstats = stmtstats + "\n"
+	}
+	return stmtstats
+}
+func getOverview(w http.ResponseWriter, r *http.Request) {
+	reports := internal.AnalyzeTables(app.conv, nil)
+	summary := internal.GenerateSummary(app.conv, reports, nil)
+	overview := writeHeading("Summary of Conversion")
+	overview = overview + summary + "\n"
+	ignored := internal.IgnoredStatements(app.conv)
+	if len(ignored) > 0 {
+		overview = overview + fmt.Sprintf("Note that the following source DB statements "+
+			"were detected but ignored: %s.\n\n",
+			strings.Join(ignored, ", "))
+	}
+	statementsMsg := ""
+	var isDump bool
+	if strings.Contains(app.driver, "dump") {
+		isDump = true
+	}
+	if isDump {
+		statementsMsg = "stats on the " + app.driver + " statements processed, followed by "
+	}
+	overview = overview + "The remainder of this report provides " + statementsMsg +
+		"a table-by-table listing of schema and data conversion details. " +
+		"For background on the schema and data conversion process used, " +
+		"and explanations of the terms and notes used in this " +
+		"report, see HarbourBridge's README.\n\n"
+	if isDump {
+		overview = overview + writeStmtStats(app.driver, app.conv)
+	}
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(overview)
+}
+
+type severity int
+
+const (
+	warning severity = iota
+	note
+)
+
+func getTypeMap(w http.ResponseWriter, r *http.Request) {
+	if app.conv == nil || app.driver == "" {
+		http.Error(w, fmt.Sprintf("Schema is not converted or Driver is not configured properly. Please retry converting the database to spanner."), http.StatusNotFound)
+		return
+	}
+	var editTypeMap map[string][]typeIssue
```

Comment:
Why is this editTypeMap?
What does the "edit" denote?

Suggest just replacing with typeMap.
And later, when we filter the map, use filteredTypeMap.

Reply:
updated to typeMap